label-application-rule-config-service-api/src/main/proto/org/hypertrace/label/application/rule/config/service/v1/label_application_rule.proto

```diff
@@ -0,0 +1,98 @@
+syntax = "proto3";
+
+package org.hypertrace.label.application.rule.config.service.v1;
+
+option java_multiple_files = true;
+
+message LabelApplicationRule {
+  string id = 1;
+  LabelApplicationRule data = 2;
+}
+
+message CreateLabelApplicationRule {
+  LabelApplicationRule data = 2;
+}
+
+message LabelApplicationRuleData {
+  string name = 1;
+  string description = 2;
```

Comment:
Not sure what our current requirements are, but I find we never really use description  and name is often just a projection of the data inside (e.g. "Apply 'Sensitive' Label"). If we don't have a requirement for user-specified name and description, I'd drop em. At least description, since if we're allowing a custom name that serves as the description.

Reply:
Yes, description is generally not used. I will remove it. I will be keep the name so that user can give the rule a relevant name.